### PR TITLE
Soup Drinking (and custom soups!)

### PR DESCRIPTION
**Pull request by SabreML**
## What Does This PR Do

1. Changes the sound for eating a bowl of soup to be a drinking sound, rather than crunching.
2. Adds 20 custom soup bowls to the Chef's vendor.
3. Makes a couple of things improper.

## Why It's Good For The Game

1. Too many croutons.
2. Custom soups!
3. `"That's Plate."` vs `"That's the plate."`

## Changelog
:cl:
add: Added 20 soup bowls to the Chef's vendor, for making custom soups!
tweak: Changed the sound for eating soup to be slurping rather than crunching.
spellcheck: Made plates and snack bowls improper, so no more 'That's Plate'.
/:cl:

### CODE REVIEW

**Review by dearmochi — APPROVED** [review]
I'M AT SOUP

**Comment by Mitchs98** [comment]
What do you mean you don't crunch your soup?

**Comment by Carthusia** [comment]
SOUP! SOUP! SOUP! SOUP!

I love soup

**Review by S34NW — APPROVED** [review]
so many soup

File: code/modules/food_and_drinks/recipes/recipes_microwave.dm [hunk 1]
```diff
@@ -227,30 +227,30 @@
...
 /datum/recipe/microwave/nettlesoup
 	reagents = list("water" = 10)
 	items = list(/obj/item/reagent_containers/food/snacks/egg, /obj/item/grown/nettle/basic, /obj/item/reagent_containers/food/snacks/grown/potato)
-	result = /obj/item/reagent_containers/food/snacks/nettlesoup
+	result = /obj/item/reagent_containers/food/snacks/soup/nettlesoup
 
 /datum/recipe/microwave/wishsoup
 	reagents = list("water" = 20)
-	result= /obj/item/reagent_containers/food/snacks/wishsoup
+	result= /obj/item/reagent_containers/food/snacks/soup/wishsoup
```
Comment (AffectedArc07) [inline code/modules/food_and_drinks/recipes/recipes_microwave.dm:245]:
```suggestion
	result = /obj/item/reagent_containers/food/snacks/soup/wishsoup
```

**Review by AffectedArc07 — CHANGES_REQUESTED** [review]
Yes I am this picky

**Comment by LordShekelstein** [comment]
Would you not be able to add some kind of "soup bowl" crafting recipe from wood or metal? Seems rather annoying to have to pester cargo everytime I want to make a custom soup because Gary Greytide threw his bowl away when he was done too many times. Or will there be some kind of implementation to clean the dirty soup bowls we have in-game? Overall, I'm not going to object to more food, this is just a small nitpicky thing.

**Comment by SabreML** [comment]
> Would you not be able to add some kind of "soup bowl" crafting recipe from wood or metal? Seems rather annoying to have to pester cargo everytime I want to make a custom soup because Gary Greytide threw his bowl away when he was done too many times. Or will there be some kind of implementation to clean the dirty soup bowls we have in-game? Overall, I'm not going to object to more food, this is just a small nitpicky thing.

The only way currently to get more soup bowls is from the vendor and leftover bowls after someone's eaten the soup.
I *could* add a crafting recipe or something along those lines, but I figured I'd wait and see if it's actually needed after it's merged. Could be that barely anyone makes soup at all, so 20 bowls is more than enough.

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**